test/cmdlineTests.sh

```diff
@@ -38,7 +38,20 @@ source "${REPO_ROOT}/scripts/common.sh"
 source "${REPO_ROOT}/scripts/common_cmdline.sh"
 
 AUTOUPDATE=false
-[[ $1 == --update ]] && AUTOUPDATE=true && shift
+no_smt=false
+while [[ $# -gt 0 ]]
+do
+    case "$1" in
+        --update)
+            AUTOUPDATE=true
```

Comment:
If you're using lowercase for your new variable, please also adjust the other one:
```suggestion
            autoupdate=true
```

Reply:
done